qcodes/instrument_drivers/american_magnetics/AMI430.py

```diff
@@ -185,14 +233,23 @@ def _can_start_ramping(self):
 
         return False
 
-    def set_field(self, value, *, perform_safety_check=True):
+    def set_field(self, value: float, perform_safety_check: bool=True) ->None:
```

Comment:
Did you intentionally remove the * that makes the following parameters keyword only?

Reply:
Yeah I did, as I do not get the point of the practice.

Reply:
Here it's imho not that important but it makes sense for functions with many arguments and especially more than one non-keyword argument